apps/microsoft-teams/frontend/src/components/config/AccessSectionCard/AccessSectionCard.spec.tsx

```diff
@@ -0,0 +1,64 @@
+import AccessSectionCard from './AccessSectionCard';
+import { describe, expect, it, vi } from 'vitest';
+import { render, screen } from '@testing-library/react';
+import { mockParameters } from '@test/mocks';
+import { accessSection } from '@constants/configCopy';
+
+describe('AccessSectionCard component', () => {
+  it('displays correct copy when authorized', () => {
+    const { unmount } = render(
+      <AccessSectionCard
+        parameters={mockParameters}
+        loginInProgress={false}
+        logoutInProgress={false}
+        handleLogin={vi.fn()}
+        handleLogout={vi.fn()}
+      />
+    );
+
+    expect(screen.getByText(mockParameters.orgName)).toBeTruthy();
+    expect(screen.getByAltText('logo')).toBeTruthy();
+    unmount();
+  });
+
+  it('displays correct copy when unauthorized', () => {
```

Comment:
Should this be ... when ~~un~~ authorized

Reply:
Yes thanks for the catch! Should be `displays correct copy when authorized with no org details`